frontend/src/pages/RegisterPage/RegisterPage.tsx

```diff
@@ -1,7 +1,522 @@
+import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider"
+import CameraIcon from "./components/Icon/CameraIcon"
+import { DatePicker } from "@mui/x-date-pickers/DatePicker"
+import dayjs from "dayjs"
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs"
+import { styled } from "@mui/material/styles"
+import { MenuItem, Select, SelectChangeEvent } from "@mui/material"
+import { useState } from "react"
+import { Gender, UserSchema } from "./types/RegisterType"
+import FortuneTellerRegisterAlert from "./components/FortuneTellerRegisterAlert"
+import ConfirmAlert from "./components/ConfirmAlert"
+import FailedAlert from "./components/FailedAlert"
+
+const today = dayjs()
+const CustomizedDatePicker = styled(DatePicker)`
+  input {
+    font-family: "Prompt", "sans-serif";
+    color: #ffffff;
+    font-size: 22px;
+    margin: 0px 14px;
+    padding: 10px;
+  }
+}
+`
+const CustomizedSelect = styled(Select)`
+  border-radius: 10px;
+
+  div {
+    font-family: "Prompt", "sans-serif";
+    color: #ffffff;
+    font-size: 22px;
+    margin-top: 6px;
+    margin-left: 16px;
+    padding: 10px;
+    border-radius: 100px;
+  }
+
+  ul {
+    border-radius: 100px;
+  }
+`
+
+const CustomizedMenuItem = styled(MenuItem)`
+  font-family: "Prompt", "sans-serif";
+  color: #3B3B3B;
+  font-size: 22px;
+}
+`
+
 export default function RegisterPage() {
+  const [formValues, setFormValues] = useState<UserSchema>({})
```

Comment:
empty object is not even a UserSchema

Reply:
คือฉันจะค่อย ๆ เพิ่มสิ่งนี้ให้มันคล้าย ๆ UserSchema ฉันควรจะประกาศเป็น type อะไร

Reply:
type นี้แหละ แต่อาจจะประกาศ default value ให้มันไปก่อน ง่ายสุดก็ {} as UserSchema มั้ง น่าจะให้ zero-value ของ ทุก type มา